File: frontend/lib/models/ingrediente.dart
class Ingrediente {
  int? id;
  int? qtd;
  String? tipo;
  String? produto;
  int? receitaId;

  Ingrediente({this.id, this.qtd, this.tipo, this.produto, this.receitaId});

  Ingrediente.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    qtd = json['qtd'];
    tipo = json['tipo'];
    produto = json['produto'];
    receitaId = json['receitaId'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['id'] = id;
    data['qtd'] = qtd;
    data['tipo'] = tipo;
    data['produto'] = produto;
    data['receitaId'] = receitaId;
    return data;
  }
}

File: backend/Models/Ingrediente.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace backend.Models
{
    public class Ingrediente
    {
        public int Id { get; set; }
        [Column(name:"Quantidade")]
        public double Qtd { get; set; }
        public string Tipo { get; set; } = String.Empty;
        [Column(name:"Nome do Produto")]
        public string Produto { get; set; } = String.Empty;
        public int ReceitaId { get; set; }
    }
}